packages/sidecar/src/main.ts

```diff
@@ -337,6 +362,8 @@ function startServer(
         server.listen(port);
         logger.info(`Port ${port} in use, retrying...`);
       }, 5000);
+    } else {
+      captureException(e);
```

Comment:
h: Catching errors is very tricky in CLI environments. They will include user paths in stack traces. In the wizard and in our bundler plugins, we basically only capture very generic error messages instead of the full error object to avoid leaking any user data into Sentry.